src/core/meshcop/secure_transport.cpp

```diff
@@ -425,7 +425,6 @@ Error SecureTransport::Setup(bool aClient)
 int SecureTransport::SetApplicationSecureKeys(void)
 {
     int rval = 0;
-
```

Comment:
Keep the newline after var declaration? (OT style guide mentions newline after var declerations).

Reply:
fixed.